src/main.cpp

```diff
@@ -41,7 +42,7 @@ int main(int argc, char** argv) {
     TableUnitConf provenance_tu = TableUnitConf();
 
     bool hopsworks = true;
-    std::string elastic_index = "projects";
+    std::string elastic_index = "all";
```

Comment:
i guess this is a typo, i guess you meant to change reindex_of to all

Reply:
fixed... sry... typo indeed